ui/ui-components/pages/model/[modelId]/property/[propertyId]/edit.tsx

```diff
@@ -197,7 +197,7 @@ export default function Page(props) {
 
     _localFilters.push({
       key: filterOptions[0].key,
-      op: "exists",
+      op: filterOptions[0].ops[0],
```

Comment:
There will always be at least one available here, yeah? Just making sure we don't run into issues trying to access undefined indexes

Reply:
All property types and plugins support, at the very least, equals, so yes!  Could also just statically set it to "equals"